content/_en/policy/rules-of-use.md

```diff
@@ -0,0 +1,157 @@
+
+  2. Customer Support
+
+Customer support and help desk services are always available through [https://login.gov/contact/](https://login.gov/contact/). When you submit for customer support, we will use any information you provide to address your question or comment, and may use your feedback to improve our service or for other purposes as we see fit, such as for marketing. In doing so, we will never reveal your personal information outside of the login.gov service except as required by applicable law or as stated elsewhere in these terms.
+
+# 8. Responsibilities of Our Partners
+
+The login.gov service connects the public to federal agencies more easily by allowing users to access multiple government programs with one login.gov account.
+
+Partner agencies select the information they require from the list of attributes we support. If we don&#39;t already have it, we request that information from you; validate it, and ask for your consent to share it with the partner agency.Partners can request a variety of information such as name, address, birth date, and phone number. Partners are required to protect this information in compliance with federal law and policy.
+
+During the process of becoming a partner on the login.gov service, we collect information from the agency about the type of access they are providing and, from that information, come to agreement with the partner on the risk of the application and the need for personal information. Based on this, we determine the level of information the partner may receive.
+
+We will never share information with a partner without your explicit consent. You may revoke this consent at any time, and we will revoke the partner&#39;s access to the information. The partner may retain it subject to their data retention policies. The login.gov service does not collect any information you provide directly to the partner.
+
+Prospective partners and other interested parties can find more information about the partner onboarding process [here](https://developers.login.gov/).
+
+# 9. Fees
+
+There are no fees charged to the user for registering or using the login.gov service. There is no fee associated with connecting to our partners, though the partners themselves may charge fees for some of their services, such as making a reservation, purchasing an item, or applying for a service.
+
+# 10. Representations, Warranties and Liabilities
+
+Disclaimer of Warranties
+
+Login.gov services are provided &quot;as is&quot; and on an &quot;as-available&quot; basis. GSA hereby disclaims all warranties of any kind, express or implied, including without limitation the warranties of merchantability, fitness for a particular purpose, and non-infringement. GSA makes no warranty that the services will be error free or that access thereto will be continuous or uninterrupted.
```

Comment:
let's just use plain quotes since markdown will smartify them for us?

```suggestion
Login.gov services are provided "as is" and on an "as-available" basis. GSA hereby disclaims all warranties of any kind, express or implied, including without limitation the warranties of merchantability, fitness for a particular purpose, and non-infringement. GSA makes no warranty that the services will be error free or that access thereto will be continuous or uninterrupted.
```